### DISCUSSION TITLE
Generating a workflow trigger URL with recipe

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#10828] [Q&A]
Is it possible to generate a workflow trigger URL with recipe?

**Comment by deanmarcussen** [comment]
Yes you can use the `TokenMethodProvider`  and `getUrlPrefix` global method

https://github.com/OrchardCMS/OrchardCore/blob/5be560147b2a8d44cc12b91a3e6201e6e9a8d0a0/src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/SignalMethodProvider.cs#L38

https://docs.orchardcore.net/en/latest/docs/reference/modules/Scripting/#content-orchardcorecontents

**Reply by DotCat1985** [reply]
How can I invoke `getUrlPrefix` global method inside a **Liquid** page?

**Reply by jptissot** [reply]
I believe that this is available on the Request object. https://docs.orchardcore.net/en/latest/docs/reference/modules/Liquid/#request
